ui/app/AppLayouts/Wallet/views/SavedAddressesView.qml

```diff
@@ -107,7 +108,10 @@ Item {
         anchors.left: parent.left
         visible: listView.count > 0
         spacing: 5
-        model: RootStore.savedAddresses
+        model: SortFilterProxyModel {
+            sourceModel: RootStore.savedAddresses
+            sorters: RoleSorter { roleName: "createdAt"; sortOrder: Qt.DescendingOrder }
```

Comment:
Yay, much better than sorting in the backend :100:

Reply:
Isn't this at the end of the day a backend sort? 😉

Reply:
Well yes :) But this is parametrized sort that can be different in different QML files, if needed ;)